### PR TITLE
tofuenv 1.0.6 (new formula)

### PR DESCRIPTION
**Pull request by kvendingoldo**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/tofuenv.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Tofuenv < Formula
+  desc "OpenTofu version manager inspired by tfenv"
+  homepage "https://github.com/tofuutils/tofuenv"
+  url "https://github.com/tofuutils/tofuenv/archive/refs/tags/v1.0.5.tar.gz"
```
Comment (chenrui333) [inline Formula/t/tofuenv.rb:4]:
```suggestion
  url "https://github.com/tofuutils/tofuenv/archive/refs/tags/v1.0.6.tar.gz"
```

File: Formula/t/tofuenv.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  sha256 "afb8e46c756b2921f0d95d02dee32ebdd7e5b26c466e8719add02bd25038c77c"
+  license "MIT"
+  head "https://github.com/tofuutils/tofuenv.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```
Comment (chenrui333) [inline Formula/t/tofuenv.rb:12]:
```suggestion
```

File: .github/autobump.txt [hunk 3]
```diff
@@ -2082,6 +2082,7 @@ tmux-xpanes
 tmuxp
 tmx
 toast
+tofuenv
```
Comment (chenrui333) [inline .github/autobump.txt:2085]: this would cover the release autoupdate part.

**Comment by chenrui333** [comment]
@kvendingoldo if you want, you can also do tap migration after this PR merge.

**Comment by chenrui333** [comment]
make sense, let me move the conflicts into a followup pr.

**Comment by kvendingoldo** [comment]
> if you want, you can also do tap migration after this PR merge.
I think that we'll do that, thank you for fixing issues in Formulas. I don't understand one thing, how can you explain it to me, pls. Looks like that software that are building from the homebrew-core should be build locally before installation that brew does. But in this case, it requires some dependencies like golang in our case. Is it ok?

**Comment by chenrui333** [comment]
> But in this case, it requires some dependencies like golang in our case. Is it ok?

yeah, it would be just build dependency.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**